workflow-ui/modal-android/src/main/java/com/squareup/workflow1/ui/modal/ModalContainer.kt

```diff
@@ -155,6 +179,14 @@ public abstract class ModalContainer<ModalRenderingT : Any> @JvmOverloads constr
       }
     }
 
+    internal fun dismiss() {
```

Comment:
```suggestion
    /**
     * Call this instead of calling `dialog.dismiss()` directly – this method ensures that the modal's
     * [WorkflowLifecycleOwner] is destroyed correctly.
     */
    internal fun dismiss() {
```